src/Pages/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,139 @@
+                        style={{
+                            borderColor:
+                                theme.colors.gray[colorMode === "dark" ? 8 : 1],
+                        }}
+                    >
+                        {[...HYMNALS_CONFIG]
+                            .sort((a, b) => (a.title < b.title ? -1 : 1))
+                            .map((value) => (
+                                <Link
+                                    key={value.key}
+                                    className={styles.link}
+                                    to={`/songs/${value.key}/1`}
+                                >
+                                    <NavLink
+                                        label={value.title}
+                                        description={value.language}
+                                    />
+                                </Link>
+                            ))}
+                    </div>
+                </div>
+            </section>
+            <section className={styles.featuresContainer} id="features">
+                <div className={styles.features}>
+                    <h2>Features</h2>
+                    <div className={styles.featuresList}>
+                        <Feature
+                            icon={<IconLanguage />}
+                            title="Multiple Language Support"
+                            subtext="Access hymnals in various languages"
+                        />
+                        <Feature
+                            icon={<IconArrowsMove />}
+                            title="Easy Navigation"
+                            subtext="Quickly find and browse through hymns"
+                        />
+                        <Feature
+                            icon={<IconSearch />}
+                            title="Search Functionality"
+                            subtext="Effortlessly search for specific hymns or keywords"
+                        />
+                        <Feature
+                            icon={<IconDeviceMobile />}
+                            title="Responsive Design"
+                            subtext="Enjoy a seamless experience on any device"
+                        />
+                    </div>
+                </div>
+            </section>
+            <Divider className={styles.divider} />
+            <footer>
+                <div className={styles.footerContents}>
+                    <p>Made with ♥ by Faith Unfeigned</p>
+                    <div className={styles.icons}>
+                        <a
+                            href="https://github.com/Faith-UnFeigned/cis-web"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            <IconBrandGithub />
+                        </a>
+                    </div>
+                </div>
+            </footer>
+        </div>
+    );
+}
+
+function Feature({
```

Comment:
Let's make sure to have only one component per file it will be easier to maintain and easier to find things.

Reply:
Okay, no problem